src/test/java/com/fasterxml/jackson/dataformat/yaml/SimpleParseTest.java

```diff
@@ -325,4 +325,38 @@ public void testObjectWithNumbers() throws Exception
 
         jp.close();
     }
+
+    public void testNulls() throws Exception
+    {
+        String YAML = "nulls: [!!null \"null\" ]";
+        JsonParser jp = YAML_F.createParser(YAML);
+
+        assertToken(JsonToken.START_OBJECT, jp.nextToken());
+        assertToken(JsonToken.FIELD_NAME, jp.nextToken());
+        assertEquals("nulls", jp.getCurrentName());
+        assertToken(JsonToken.START_ARRAY, jp.nextToken());
+        assertToken(JsonToken.VALUE_NULL, jp.nextToken());
+        assertToken(JsonToken.END_ARRAY, jp.nextToken());
+        assertToken(JsonToken.END_OBJECT, jp.nextToken());
+        assertNull(jp.nextToken());
+
+        jp.close();
+    }
+
+    public void testTildeNulls() throws Exception
+    {
+        String YAML = "nulls: [~ ]";
```

Comment:
I put a bunch of spec links into a comment, didn't you see them?

On Thu, Dec 24, 2015 at 2:38 PM, Tatu Saloranta <notifications@github.com>
wrote:

> In
> src/test/java/com/fasterxml/jackson/dataformat/yaml/SimpleParseTest.java
> <https://github.com/FasterXML/jackson-dataformat-yaml/pull/51#discussion_r48424774>
> :
>
> > +
> > +        assertToken(JsonToken.START_OBJECT, jp.nextToken());
> > +        assertToken(JsonToken.FIELD_NAME, jp.nextToken());
> > +        assertEquals("nulls", jp.getCurrentName());
> > +        assertToken(JsonToken.START_ARRAY, jp.nextToken());
> > +        assertToken(JsonToken.VALUE_NULL, jp.nextToken());
> > +        assertToken(JsonToken.END_ARRAY, jp.nextToken());
> > +        assertToken(JsonToken.END_OBJECT, jp.nextToken());
> > +        assertNull(jp.nextToken());
> > +
> > +        jp.close();
> > +    }
> > +
> > +    public void testTildeNulls() throws Exception
> > +    {
> > +        String YAML = "nulls: [~ ]";
>
> If you can add a suggested change here I'll cut'n paste. A link to spec
> would be all I need if you know it off-hand?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FasterXML/jackson-dataformat-yaml/pull/51/files#r48424774>
> .
>